apps/system/build/build.js

```diff
@@ -146,6 +146,35 @@ SystemAppBuilder.prototype.enableFirefoxSync = function(options) {
   preprocessor.execute(options, 'FIREFOX_SYNC', fileList);
 };
 
+SystemAppBuilder.prototype.disableBluetoothFeatures = function(options) {
+  var fileList = {
+    process: [
+      ['index.html'],
+      ['js', 'quick_settings.js']
+    ],
+    remove: {
+      'ifdef': [
+        ['js', 'bluetooth.js'],
+        ['js', 'bluetooth_core.js'],
+        ['js', 'bluetooth_headphone.js'],
+        ['js', 'bluetooth_icon.js'],
+        ['js', 'bluetooth_transfer.js'],
+        ['js', 'bluetooth_transfer_icon.js'],
+        ['js', 'bluetooth_transfer_v2.js'],
+        ['test', 'unit', 'bluetooth_core_test.js'],
+        ['test', 'unit', 'bluetooth_core_v2_test.js'],
+        ['test', 'unit', 'bluetooth_headphone_icon_test.js'],
+        ['test', 'unit', 'bluetooth_icon_test.js'],
+        ['test', 'unit', 'bluetooth_test.js'],
+        ['test', 'unit', 'bluetooth_transfer_icon_test.js'],
+        ['test', 'unit', 'bluetooth_transfer_test.js'],
+        ['test', 'unit', 'bluetooth_v2_test.js']
+      ]
```

Comment:
Hmm, I figured "disabling UI aspect of platform feature X", as we're trying to do here, would be pretty difficult as platform features are relied upon in other features, bluetooth for NFC initiated transfer as you've mentioned, for example.   It raises questions about whether we should think of graceful degradation in FxOS - afaik Gaia has always had the luxury of knowing what platform features are supported and relying on them, is this still a fair assumption to make.. I'm rambling, will take this into an email.